### PR TITLE
Transaction Update

### PR DESCRIPTION
**Pull request by colinbird**
Support manual update of transaction result, amount, currency.

### CODE REVIEW

File: openapi/components/schemas/Transactions/TransactionUpdate.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+type: object
+description: Update a Transaction manually to completed status with given result with optional currency and amount.
+required:
+  - result
+properties:
+  result:
+    description: Transaction result.
+    type: string
+    readOnly: true
```
Comment (colinbird) [inline openapi/components/schemas/Transactions/TransactionUpdate.yaml:9]:
```suggestion
```

File: openapi/paths/transactions@{id}@update.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  description: Update a Transaction manually to completed status with given result with optional currency and amount.
+  requestBody:
+    content:
+      application/json:
+        schema:
+          $ref: ../components/schemas/Transactions/TransactionUpdate.yaml
+    required: true
+  responses:
+    '200':
+      description: Transaction was not updated as paramters matched stored data.
```
Comment (adamaltman) [inline openapi/paths/transactions@{id}@update.yaml:18]:
```suggestion
      description: Transaction was not updated as parameters matched stored data.
```

**Review by removed-account-123 — DISMISSED** [review]
LGTM 👍

**Comment by iamdevice** [comment]
php please review

**Review by adamaltman — DISMISSED** [review]

**Review by XaosSintez — DISMISSED** [review]

**Review by removed-account-123 — DISMISSED** [review]

**Comment by rensftw** [comment]
@iamdevice Shouldn't we also add `transactions@{id}@update.yaml` in `core.yaml`?

**Review by XaosSintez — APPROVED** [review]

**Review by rensftw — APPROVED** [review]

**Pull request merged**